README.md

```diff
@@ -0,0 +1,54 @@
+[![Research software impact](http://depsy.org/api/package/pypi/khmer/badge.svg)](http://depsy.org/package/python/khmer)
+![Supported Python versions](https://img.shields.io/pypi/pyversions/khmer.svg)
```

Comment:
This, as well as the GitHub language breakdown, is IMO sufficient for language disclosure a. la. #1250.